README.md

```diff
@@ -67,6 +82,10 @@ Zivid Samples are distributed under the [BSD license](LICENSE).
 [CaptureVis3D-url]: source/Applications/Basic/Visualization/CaptureVis3D/CaptureVis3D.cs
 [ReadIterateZDF-url]: source/Applications/Basic/FileFormats/ReadIterateZDF/ReadIterateZDF.cs
 [ZDF2PLY-url]: source/Applications/Basic/FileFormats/ZDF2PLY/ZDF2PLY.cs
-[HandEyeCalibration-url]: source/Applications/Advanced/HandEyeCalibration/HandEyeCalibration.cs
+[HandEyeCalibrationReadme-url]: source/Applications/Advanced/HandEyeCalibration/README.md
+[HandEyeCalibration-url]: source/Applications/Advanced/HandEyeCalibration/HandEyeCalibration/HandEyeCalibration.cs
```

Comment:
Can you separate out this line of code from this and add it to the previous commit. This is basically showing moving the sample.

Reply:
done